cfpb/cfgov-refresh
# Mega menu: Update mobile hover state

## Changes

- Left align all items in the mobile menu, including global CTA.
- Add green hover to mobile menu items.
- Convert global CTA link to a native link in the menu.
- Rename menu `_classes` helper to `_content_classes` as works on `o-mega-menu_content`
- Add `o-mega-menu_content-list-group` container.

## Testing

1. Pull branch and build.
2. Visit http://localhost:8000/policy-compliance/ and resize to mobile.
3. Note currently selected item and hover state.
4. Desktop should be unchanged from production.

## Screenshots

<img width="758" alt="Screen Shot 2020-03-13 at 5 26 07 PM" src="https://user-images.githubusercontent.com/704760/76661398-b87a4a80-6551-11ea-824a-45e04d1b535e.png">

<img width="828" alt="Screen Shot 2020-03-13 at 5 42 43 PM" src="https://user-images.githubusercontent.com/704760/76661648-5d952300-6552-11ea-9fb8-6153d1040c28.png">
